src/amo/components/SearchResult/index.js

```diff
@@ -164,7 +164,10 @@ export class SearchResultBase extends React.Component<InternalProps> {
     if (showSummary) {
       const summaryProps = {};
       if (addon) {
-        summaryProps.dangerouslySetInnerHTML = sanitizeHTML(addon.summary);
+        summaryProps.dangerouslySetInnerHTML = sanitizeHTML(
+          addon.summary ? addon.summary.content : '',
+        );
+        summaryProps.lang = addon.summary ? addon.summary.locale : null;
```

Comment:
This should be tested.

Reply:
I couldn't see the test for this - can you point to it (or add it)?

Reply:
(it's confusing that the test isn't in a file matching the name of the component, but...) I meant the case when `summary` isn't set and the code is doing something slightly different.

Reply:
I see, when the summary doesn't exist, there was no test for it. I added it in the same file below the test in the most recent commit